share/wake/lib/system/job_cache_runner.wake

```diff
@@ -60,7 +60,7 @@ export def mkJobCacheRunner (hashFn: RunnerInput => Result String Error) (wakero
     def job_cache_add str = prim "job_cache_add"
 
     def run (job: Job) (input: RunnerInput): Result RunnerOutput Error =
-        def (RunnerInput label cmd vis env dir stdin _ prefix _ _) = input
+        def (RunnerInput label cmd vis env dir stdin _ prefix _ _ _ _) = input
```

Comment:
Is it valid not to use `fnInputs` or `fnOutputs`? Is that not part of the runner's contract with the `Plan` and `Job`?
Or are we not using them here because this  function calls another `run` function that does call those input/output functions?

Reply:
> Is it valid not to use fnInputs or fnOutputs?

Nope, it must always be called at least once. In this case, the cache runners don't call it because the runner they are wrapping is expected to call it